From e4dae226fe09e41a787aebf8acccd72d819cd690 Mon Sep 17 00:00:00 2001
From: Ferry Toth <ftoth@exalondelft.nl>
Date: Thu, 19 Sep 2019 22:19:32 +0200
Subject: [PATCH] u-boot: Fix path to merge_config.sh

Current u-boot recipe in warrior fails during do_configure because
it cannot find the merge_config.sh. Fix the path

Suggested-by: nishantpoorswani <nishantpoorswani@gmail.com>
Signed-off-by: Ferry Toth <ftoth@exalondelft.nl>
---
 meta/recipes-bsp/u-boot/u-boot.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta/recipes-bsp/u-boot/u-boot.inc b/meta/recipes-bsp/u-boot/u-boot.inc
index 96c1f94011..390b92850d 100644
--- a/meta/recipes-bsp/u-boot/u-boot.inc
+++ b/meta/recipes-bsp/u-boot/u-boot.inc
@@ -85,7 +85,7 @@ do_configure () {
         else
             oe_runmake -C ${S} O=${B} oldconfig
         fi
-        merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
+        ${S}/scripts/kconfig/merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
         cml1_do_configure
     fi
 }
-- 
2.20.1

